src/qibolab/dummy/platform.py

```diff
@@ -86,17 +64,16 @@ def create_dummy(with_couplers: bool = True):
         qubit.flux = DcChannel(flux_name)
         configs[flux_name] = DcConfig(**component_params[flux_name])
 
-    if with_couplers:
-        for c, coupler in couplers.items():
-            flux_name = f"coupler_{c}/flux"
-            coupler.flux = DcChannel(flux_name)
-            configs[flux_name] = DcConfig(**component_params[flux_name])
+    for c, coupler in couplers.items():
+        c = c[1:]
+        flux_name = f"coupler_{c}/flux"
+        coupler.flux = DcChannel(flux_name)
+        configs[flux_name] = DcConfig(**component_params[flux_name])
 
     instruments = {instrument.name: instrument, twpa_pump.name: twpa_pump}
     instruments = load_instrument_settings(runcard, instruments)
-    name = "dummy_couplers" if with_couplers else "dummy"
     return Platform(
-        name,
+        "dummy",
```

Comment:
Mostly irrelevant, but another option that we typically use in qibolab_platforms_qrc to avoid name duplication (and potential errors) is
```suggestion
        FOLDER.name,
```